src/System.Runtime/tests/CompareToTests.netstandard1.7.cs

```diff
@@ -0,0 +1,165 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Xunit;
+
+namespace System.Tests
+{
+    public static partial class CompareToTests
+    {
+        [Fact]
+        public static void FromBoolean()
+        {
+            object testValue = false;
+            object testValue2 = true;
+            object wrongValue = "hello";
+            bool val1 = false;
+            bool val2 = true;
+            Assert.Equal(0, val1.CompareTo(testValue));
```

Comment:
Personally, I think it would be easier to read these things if we didn't define extra variables and just used the values inline:

```csharp
Assert.Equal(0, false.CompareTo((object)false));
Assert.True(0 > false.CompareTo((object)true));
Assert.True(0 < true.CompareTo((object)false);
Assert.Throws<ArgumentException>(() => false.CompareTo("hello"));
```

The above also shows that we don't have a case that does `true.CompareTo((object)true)` and asserts it's equal to 0.